packages/core/useInfiniteScroll/index.ts

```diff
@@ -11,6 +11,20 @@ export interface UseInfiniteScrollOptions extends UseScrollOptions {
    * @default 0
    */
   distance?: number
+
+  /**
+   * The direction in which to listen the scroll.
+   *
+   * @default 'bottom'
+   */
+  direction?: 'top' | 'bottom'
+
+  /**
+   * Whether to keep the current scroll position when loading more items.
+   *
+   * @default false
+   */
+  keepScrollPosition?: boolean
```

Comment:
I think `preserveScrollPosition` might be a better name

Reply:
You're right 👍